src/Yasumi/Provider/Austria/Salzburg.php

```diff
@@ -0,0 +1,75 @@
+<?php declare(strict_types=1);
+/**
+ * This file is part of the Yasumi package.
+ *
+ * Copyright (c) 2015 - 2019 AzuyaLabs
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @author Sacha Telgenhof <me@sachatelgenhof.com>
+ */
+
+namespace Yasumi\Provider\Austria;
+
+use DateTime;
+use Yasumi\Exception\InvalidDateException;
+use Yasumi\Exception\UnknownLocaleException;
+use Yasumi\Holiday;
+use Yasumi\Provider\Austria;
+
+/**
+ * Provider for all holidays in Salzburg (Austria).
+ *
+ * @link https://en.wikipedia.org/wiki/Salzburg_(state)
+ */
+class Salzburg extends Austria
+{
+    /**
+     * Code to identify this Holiday Provider. Typically this is the ISO3166 code corresponding to the respective
+     * country or sub-region.
+     */
+    public const ID = 'AT-5';
+
+    /**
+     * Initialize holidays for Salzburg (Austria).
+     *
+     * @throws InvalidDateException
+     * @throws \InvalidArgumentException
+     * @throws UnknownLocaleException
+     * @throws \Exception
+     */
+    public function initialize(): void
+    {
+        parent::initialize();
+
+        // Add custom holidays.
+        $this->calculateStRupertsDay();
+    }
+
+    /**
+     * Saint Rupert's Day.
+     *
+     * Rupert of Salzburg was Bishop of Worms as well as the first Bishop of
+     * Salzburg and abbot of St. Peter's in Salzburg. He was a contemporary of
+     * the Frankish king Childebert III and is venerated as a saint in the
+     * Roman Catholic and Eastern Orthodox Churches. Rupert is also patron
+     * saint of the Austrian state of Salzburg.
+     *
+     * @link https://en.wikipedia.org/wiki/Rupert_of_Salzburg
+     *
+     * @throws InvalidDateException
+     * @throws \InvalidArgumentException
+     * @throws UnknownLocaleException
+     * @throws \Exception
+     */
+    private function calculateStRupertsDay(): void
```

Comment:
Can you add a condition for when this holiday was first introduced/started?

Reply:
Looks like the holiday has been started on 710-9-24. Yasumi allows years from 1000. What is the best way to proceed?

Reply:
Ah. In that case you can leave it as is then :) Perhaps just add a note in the docblock.